php/WP_CLI/Runner.php

```diff
@@ -534,6 +534,12 @@ private static function back_compat_conversions( $args, $assoc_args ) {
 					$assoc_args['orderby'] = 'post__in';
 					$assoc_args['field'] = 'url';
 					break;
+				case 'term':
+					$term_ids = array_slice( $args, 3 );
+					$args = array( 'term', 'list', $args[2] );
+					$assoc_args['include'] = implode( ',', $term_ids );
```

Comment:
@danielbachhuber Ideally we should. I digged down re term query. Only after 4.1 `'orderby'=>'include'` was introduced in `get_terms()`. I am not sure what is the best path here. But personally I think, anything we are planning to deprecate, we should do it before `1.0.0` or even earlier.